app/common/renderer/actions/SessionInspector.js

```diff
@@ -327,9 +329,40 @@ export function applyClientMethod(params) {
       window.dispatchEvent(new Event('resize'));
       return commandRes;
     } catch (error) {
+      const {inspector} = getState();
+      const isWebDriverError = error
+        .toString()
+        .includes('process is not running (probably crashed)');
```

Comment:
The message would depend on the implementation on the driver side, so this might be difficult to maintain. Let me think about this a bit

Reply:
Hi @KazuCocoa ,
In the recent push, I added a custom error message to help identify whether the session has crashed. please review it from your side and I’m happy to discuss any improvements or alternatives you might suggest.